api/console_message.go

```diff
@@ -0,0 +1,16 @@
+package api
+
+// ConsoleMessage represents a page console message.
+// · Args represent the list of arguments passed to a console function call.
+// · Page is the page that produced the console message, if any.
+// · Text represents the text of the console message.
+// · Type is the type of the console message. It can be one of 'log',
+// 'debug', 'info', 'error', 'warning', 'dir', 'dirxml', 'table', 'trace',
+// 'clear', 'startGroup', 'startGroupCollapsed', 'endGroup', 'assert', 'profile',
+// 'profileEnd', 'count', 'timeEnd'.
+type ConsoleMessage struct {
+	Args []JSHandle
+	Page Page
+	Text string
+	Type string
+}
```

Comment:
This comment will render the docs like this:

<img width="844" alt="Screenshot 2023-08-28 at 13 01 05" src="https://github.com/grafana/xk6-browser/assets/621232/612d911b-faa9-416d-b588-bcf4eff21c9f">

I believe the following suggestion makes it better readable on the docs and in the IDEs.

```suggestion
// ConsoleMessage represents a page console message.
type ConsoleMessage struct {
	// Args represent the list of arguments passed to a console function call.
	Args []JSHandle

	// Page is the page that produced the console message, if any.
	Page Page

	// Text represents the text of the console message.
	Text string

	// Type is the type of the console message.
	//
	// It can be one of log, debug, info, error, warning, dir, dirxml,
	// table, trace, clear, startGroup, startGroupCollapsed, endGroup,
	// assert, profile, profileEnd, count, timeEnd.
	Type string
}
```

For example:
<img width="793" alt="Screenshot 2023-08-28 at 13 04 27" src="https://github.com/grafana/xk6-browser/assets/621232/b1911336-5a5c-4707-b31c-f82e4cb0972e">

Reply:
I was doubtful what was better for Godoc as there are examples for both approaches, but this LGTM :+1: 
Changed in [eac5e96](https://github.com/grafana/xk6-browser/pull/1006/commits/eac5e96810feeb5828d47e3215532a2574d45c98).